static/bidder-info/criteo.yaml

```diff
@@ -14,3 +14,7 @@ userSync:
   # bidder directly at the email address in this file to ask about enabling user sync.
   supports:
     - redirect
+
+  redirect:
+    url: "https://ssp-sync.criteo.com/user-sync/redirect?gdprapplies={{.GDPR}}&gdpr={{.GDPRConsent}}&ccpa={{.USPrivacy}}&redir={{.RedirectURL}}&profile=230"
+    userMacro: "${CRITEO_USER_ID}"
```

Comment:
<img width="1791" alt="image" src="https://user-images.githubusercontent.com/24757781/220557847-ccfd1a6e-bd23-4fca-9e39-5668060ad04d.png">

url isn't redirecting https://ssp-sync.criteo.com/user-sync/redirect?gdprapplies=&gdpr=&ccpa=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dcriteo%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BCRITEO_USER_ID%7D&profile=230

Reply:
Hello, thanks for the review. To be able to redirect to a specific domain, we need a configuration on our side. I authorized the domain you want to redirect too and now it redirects.
I will add a comment to make sure this is clear.

Reply:
thanks @vraybaud.  Redirection is working

Reply:
working for me as well.